Editor/EditorCore/EditorToolbar.cs

```diff
@@ -67,6 +67,8 @@ void OnEnable()
             isIconMode = ProBuilderEditor.s_IsIconGui;
             this.window = ProBuilderEditor.instance;
             CalculateMaxIconSize();
+
+            MeshSelection.ClearElementSelection();
```

Comment:
Without checking, it looks like this would clear the element selection on domain reloads. Could this be resolved instead by modifying the constructor of the ProBuilderize menu action? Looking at the `ProBuilderize()` constructor it looks like it doesn't actually set `m_Enabled` when it's instantiated.